known_friends.md

```diff
@@ -5,6 +5,8 @@ To get to know how we define "adoptions" and see how to submit a feature, please
 | Data tracking by auditing data                     | Proof of concept of how to track data usage by aggregating audit-logs of different components in a single instance: The work presents a first proof of concept of how traceability of data usage can be implemented using the EDC Connector event framework and audit logging. In this PoC, the traceability of data is limited to the AWS dataplane. The EDC Connector logs which exact assets are stored with which key in the AWS bucket. With this information, data usage can be traced from the shared logs of the EDC Connector and the AWS S3 bucket. Elasticsearch was chosen as the instance to merge both logs in this project. A simple Python script takes over the analysis of the log data.                                  | [jan.peter.meyer@isst.fraunhofer.de](mailto:jan.peter.meyer@isst.fraunhofer.de)                                                              | [Link to Repo](https://github.com/FraunhoferISST/edc-data-tracebility-app)                                                                                                                                                                                            |
+| IDS Broker Extension                               | EDC connector extension that can register offered assets in form of IDS Resources to a central IDS Metadata Broker.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | [contact@sovity.de](mailto:contact@sovity.de)                                                                                                | [Link to Repo](https://github.com/sovity/edc-extensions) |
```

Comment:
Link for the IDS-Metadata Broker extension should be https://github.com/sovity/edc-extensions/tree/main/extensions/broker, right?

In addition most configuration values in the documentation are pointing to instances of the Mobility Data Space. For me it is unclear if you provide an extension to a generic [IDS Metadata Broker](https://github.com/International-Data-Spaces-Association/metadata-broker-open-core) or a MDS flavored version of it. That should be made clear, at least in the title within the table.

Reply:
Yes, please update the link, the repo has evolved differently than initially thought.
The extension should be compatible with a generic metadata broker.